actions/create-workflows/action.yaml

```diff
@@ -108,11 +96,28 @@ runs:
           eval $COMMAND
 
           cat workflow_result.txt
-          ORDER_ID=$(cat workflow_result.txt | grep -oP '\["\K\d+(?="\])') 
+          ORDER_ID=$(grep -oP '\["\K\d+(?="\])' workflow_result.txt)
+
+          if ! [[ "$ORDER_ID" =~ ^[0-9]+$ ]]; then
```

Comment:
We are under the [loop](https://github.com/Super-Protocol/sp-build-tools/pull/5/files#diff-4553d8a8998b9ae3be70f81f50ce136fe3770e88bbad0cae6b8a7d386401ca13R87) here.
There is a need for more complex logic. If one order was created and another was not, the first order needs to be canceled.